rdrf/rdrf/settings.py

```diff
@@ -53,11 +53,17 @@
 LANGUAGE_CODE = env.get("language_code", 'en')
 USE_I18N = env.get("use_i18n", True)
 
-LANGUAGES = (
-    ('ar', 'Arabic'),
-    ('de', 'German'),
-    ('en', 'English'),
-)
+# This must be a superset of LANGUAGES
```

Comment:
Originally we had one languages list - the EmailTemplate model field languages using it to filla  dropdown when we exposed to settings / env we created a situation where different deployments would trigger migration errors.
\
To avoid error I created a superset


yes I guess we could  check